client/src/containers/blog/Blog.js

```diff
@@ -1,27 +1,94 @@
-import React from 'react';
-import { Route, Switch, useRouteMatch } from 'react-router-dom';
-import LinkButton from '../../components/LinkButton';
-import BlogPost from '../blog-post/BlogPost';
+import React, { useEffect, useState } from 'react';
+import { Col, Row, Spin, Typography } from 'antd';
+import { Trans } from '@lingui/macro';
+import BlogItem from '../../components/BlogItem/BlogItem';
+import config from '../../config';
+import LoadMore from '../../components/LoadMoreArticlesLink/LoadMoreArticlesLink';
+
+const { Title, Paragraph } = Typography;
+
+const { POSTS_URL } = config;
+
+const LIMIT = 3;
 
 const Blog = () => {
-  const match = useRouteMatch();
+  const [state, setState] = useState({
+    posts: [],
+    requestError: false,
+    loading: true,
+    index: 0,
+    showMore: true,
+  });
+
+  const loadPosts = async () => {
+    try {
+      const res = await fetch(`${POSTS_URL}`);
```

Comment:
We will have to change the fetch URL to `https://api-seismicrisc.code4.ro/api/v1/posts/?ordering=-created`